Gemfile.lock

```diff
@@ -251,7 +251,7 @@ GEM
       zeitwerk (~> 2.5)
     rainbow (3.1.1)
     rake (13.0.6)
-    rbi (0.0.15)
+    rbi (0.0.16)
```

Comment:
Let's update the required version in the gemspec file as well: https://github.com/Shopify/tapioca/blob/emily/tstruct-field-tests/tapioca.gemspec#L30